code/tINIT/getINITModel2.m

```diff
@@ -1,5 +1,7 @@
 function [model, metProduction, essentialRxnsForTasks, addedRxnsForTasks, deletedDeadEndRxns, deletedRxnsInINIT, taskReport] = getINITModel2(refModel, tissue, celltype, hpaData, arrayData, metabolomicsData, removeGenes, taskFile, useScoresForTasks, printReport, taskStructure, params, paramsFT)
 % getINITModel2
+%   This function is deprecated - we suggest that you instead use ftINIT in
+%   RAVEN Toolbox together with the helpers in this repo.
```

Comment:
I have mixed feelings about this. This function is still (nearly) the original implementation of the algorithm, and is not necessarily "deprecated". Although ftINIT is faster, it uses a different approach (with additional assumptions and simplifications). So unless we can definitively show that ftINIT is a more accurate algorithm, I hesitate to "deprecate" the original tINIT.

Reply:
The question is what to write here. Should we say that there is a newer version, but not call it deprecated? I have some reason to believe that ftINIT is performing better. I'm supplying a powerpoint that shows one test I did with this function and compared it to tINIT without allowing flux in both direction and without metabolite secretion. ftINIT in this test gave the same result as the one where these were not allowed, while the standard getINITModel2 produced something that is quite different, full of gaps. The task step of ftINIT typically fills in ~13 rxns, while the same step for getINITModel2 fills in ~55 rxns. So, I think that getINITModel2 has some problems, and that the problems with ftINIT are much smaller. But I have not tested it super much.
[tINITTestInfo.pptx](https://github.com/SysBioChalmers/Human-GEM/files/8741931/tINITTestInfo.pptx)

Reply:
Ok, very good points. I think it's fine to state that there is a newer version, but would avoid using the word "deprecated", since it sometimes implies that the function will eventually be removed (which is not the intent).

I agree that some of the behavior you've seen with ftINIT (other than shorter compute time) can be interpreted as a clear improvement in performance/accuracy.

Reply:
Ok, I wrote that there is a new improved version of the function available in RAVEN, I have pushed a new commit regarding this.